### DISCUSSION TITLE
طريقة تثبيت الاختراق

### DISCUSSION THREAD

**Discussion by Abdu38hkem** [jaykali/maskphish#129] [Q&A]
كيف يمكنني معرفة انا الضحية قد وقع في الفخ

**Comment by jaykali** [comment]
لم يغير "Maskphish" عنوان URL ، ولكنه يستخدم فقط بعض تقنيات الهندسة الاجتماعية لجعل الرابط شرعيًا للضحايا واستخدام اختصار url. إذا رأيت علامة @ على عنوان URL ، فيمكن أن يكون عنوان URL مقنعًا. تتمثل أفضل الممارسات في عدم النقر فوق أي عنوان URL تم الحصول عليه من شخص لا تثق به.

(Translate by Google)
